server/metadata/namespace.go

```diff
@@ -0,0 +1,38 @@
+package metadata
```

Comment:
So instead of having it specific to the namespace, should we just have a generic class in server package `RequestMetadata` or `ContextMetadata` etc and that will have a bunch of methods to extract the values of different keys in the context that is needed in the server package. This way nothing changes in the metadata package and once you get the namespace name then sessions.go will simply follow the same process that it is doing now.

Reply:
added a wrapper RequestMetadata type.